src/test/java/se/liu/ida/hefquin/engine/wrappers/lpgwrapper/NodeMappingToBNodesImplTest.java

```diff
@@ -0,0 +1,42 @@
+package se.liu.ida.hefquin.engine.wrappers.lpgwrapper;
+
+import org.apache.jena.graph.Node;
+import org.apache.jena.graph.NodeFactory;
+import org.junit.Test;
+import se.liu.ida.hefquin.engine.wrappers.lpgwrapper.data.impl.LPGNode;
+import se.liu.ida.hefquin.engine.wrappers.lpgwrapper.impl.NodeMapping;
+import se.liu.ida.hefquin.engine.wrappers.lpgwrapper.impl.NodeMappingToBNodesImpl;
+
+import static org.junit.Assert.*;
+
+public class NodeMappingToBNodesImplTest {
+    protected final String NS = "https://example.org/";
+    protected final String NODE = "node/";
```

Comment:
You can concatenate these two into a single string and, then, move it into the one test function that uses it (i.e., no need to have this as a member variable in this class).